guacamole/src/main/java/org/glyptodon/guacamole/net/basic/extension/ExtensionManifest.java

```diff
@@ -312,4 +324,46 @@ public void setAuthProviders(Collection<String> authProviders) {
         this.authProviders = authProviders;
     }
 
+    /**
+     * Returns the path to the small favicon, relative to the root of the
+     * extension.
+     *
+     * @return The path to the small favicon.
+     */
+    public String getSmallIcon() {
+        return smallIcon;
+    }
+
+    /**
+     * Sets the path to the small favicon. This will replace the default
+     * guacamole icon.
+     *
+     * @param smallIcon The path to the small favicon.
```

Comment:
For new code, these comments should be formatted like:

```
@param smallIcon
    The path to the small favicon.
```

Same deal for `@return`. Take a peek at the other code in this file.